src/components/views/settings/tabs/user/GeneralUserSettingsTab.tsx

```diff
@@ -351,9 +357,29 @@ export default class GeneralUserSettingsTab extends React.Component<IProps, ISta
             passwordChangeForm = null;
         }
 
+        let externalAccountManagement: JSX.Element;
+        if (this.state.externalAccountManagementUrl) {
+            const externalAccountManagementText = _t(
+                '<a>Manage your account</a>.',
```

Comment:
is it really required to have the `.` not part of the link? Can we just drop the full stop and move the anchor outside of the translation?

Reply:
The latest iteration has `Manage your account at <a>%(hostname)s</a>.`.

@turt2live do you still have a concern with the latest wording?